package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@wasm-tool/rollup-plugin-rust",
   "author": "Pauan <pauanyu+github@pm.me>",
   "description": "Rollup plugin for bundling and importing Rust crates.",
-  "version": "1.0.3",
+  "version": "1.0.4",

```

Comment:
I would prefer to bump the version separately.

Reply:
I wasn't sure if I should do it or not - it was a bit of an after thought